joomla/joomla-cms
# [4.0] Add empty alt checkbox for tinyMCE dnd

Pull Request for Issue # .

### Summary of Changes
Adds the checkbox for specifying that an image is decorative (alt=""). Feature parity with the rest fields, etc


### Testing Instructions
Drag and drop an image in the tinyMCE, check or not the `No Description` checkbox (adding or not alt text) and observe the generated tag (just toggle the editor to do so)

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required
